pkg/ucauth/kratos/kratos.yml

```diff
@@ -0,0 +1,80 @@
+version: v0.4.6-alpha.1
+
+dsn: memory
+
+serve:
+  public:
+    base_url:  /.ory/kratos/public
+    cors:
+      enabled: true
+  admin:
+    base_url: http://kratos:4434/
+
+selfservice:
+  default_browser_return_url: http://one.erda.local
```

Comment:
should be replace to related path, not static domain

Reply:
Config source validation has been improved in later ory/kratos version. we are not using it.  it can be replaced by env SELFSERVICE_DEFAULT_BROWSER_RETURN_URL .